pkg/orm/orm_raw.go

```diff
@@ -369,7 +369,15 @@ func (o *rawSet) QueryRow(containers ...interface{}) error {
 							field.Set(mf)
 							field = mf.Elem().FieldByIndex(fi.relModelInfo.fields.pk.fieldIndex)
 						}
-						o.setFieldValue(field, value)
+						if fi.isFielder {
+							fd := field.Addr().Interface().(Fielder)
+							err := fd.SetRaw(value)
+							if err != nil {
+								return fmt.Errorf("set raw error:%s", err)
```

Comment:
What about using `github.com/pkg/errors`, we can change it in the future. I will check this PR this weekend.

Reply:
Sure. But I wonder what's the difference between `errors.New()` and `fmt.Errorf()` ?